libpod/container_api.go

```diff
@@ -745,3 +747,77 @@ func (c *Container) Sync() error {
 
 	return nil
 }
+
+func (c *Container) setupOCIHooks(g *generate.Generator) error {
```

Comment:
Is this ever called when we create the container? I don't think it is

Reply:
Also, this is a private function, stick is in container_internal.go

Reply:
Fixed.